### DISCUSSION TITLE
Coupling open space with porous media flow

### DISCUSSION THREAD

**Discussion by jundonglin** [PMEAL/OpenPNM#2882] [Q&A]
This is my geometric model, this is a two-dimensional pipe, the wall of the pipe is lined with a porous medium, the inlet velocity of water is about 5 meters per second, considering the viscosity, I want to simulate this flow, get the pressure distribution of the flow field, the velocity distribution. I'd like to ask you for a general simulation plan.

**Comment by jundonglin** [comment]
![1](https://github.com/PMEAL/OpenPNM/assets/151015751/8ad6219d-4080-4a44-b6bd-709e51600559)

**Comment by ma-sadeghi** [comment]
You can treat your open space channel as a series of hypothetical nodes and add it to your global network model. Of course, you'll have to deal with the connections and the conductances (channel-porous interface throats) manually.
